backend/aiconsole/core/users/user.py

```diff
@@ -25,14 +32,42 @@ def get_profile(self, email: str | None = None) -> UserProfile:
             gravatar=False,
         )
 
+    def save_avatar(self, file: BinaryIO, file_name: str | None = None, content_type: str | None = None) -> None:
+        extension = guess_extension(content_type) if content_type else None
+        if not file_name:
+            if not extension:
+                raise MissingFileName()
+            file_name = f"avatar{extension}"
+
+        file_path = self.get_avatar(file_name)
+        self._save_avatar_to_fs(file, file_path)
+
+        avatar_url = f"profile_image?img_filename={file_path.name}"
+        settings().storage.save(
+            PartialSettingsData(
+                user_profile_settings=UserProfile(avatar_url=avatar_url),
+                to_global=True,
+            ),
+        )
+
+    def get_avatar(self, img_filename: str) -> Path:
+        return self.get_avatar_folder_path() / img_filename
+
+    @staticmethod
+    def get_avatar_folder_path() -> Path:
+        avatar_folder_path = AICONSOLE_USER_CONFIG_DIR() / "avatars"
+        if not avatar_folder_path.exists():
+            avatar_folder_path.mkdir(parents=False, exist_ok=True)
```

Comment:
```suggestion
        avatar_folder_path.mkdir(parents=False, exist_ok=True)
```

Could you check if it will be enough? You types exist_ok, so I think this if statement is redundant

Reply:
Checked info, They have different methods of verifying the existence, but `exist_ok` seems to be self sufficient.